capirca/lib/paloaltofw.py

```diff
@@ -750,10 +752,27 @@ def __str__(self):
     rulebase = etree.SubElement(vsys_entry, "rulebase")
     security = etree.SubElement(rulebase, "security")
     rules = etree.SubElement(security, "rules")
+    tag = etree.Element("tag")
+
+    tag_num = 0
 
     # pytype: disable=key-error
     # pylint: disable=unused-variable
     for (header, pa_rules, filter_options) in self.pafw_policies:
+      tag_name = None
+      if header.comment:
+        comment = " ".join(header.comment).strip()
+        if comment:
+          tag_num += 1
```

Comment:
Since this is done at the beginning of the clause we'll never print a comment 0. I just want to make sure that's intentional?

Reply:
the number in the tag name is intended to start at 1.